Apps/Src/ReadSwitches.c

```diff
@@ -91,4 +91,5 @@ static void UpdateLights() {
     if(rightblink==0){
         Lights_Set(RIGHT_BLINK,OFF);
     }
+    
 }
```

Comment:
No that important but should probably have newlines at the end of all library files.